C/CMake/build_tarballs.jl

```diff
@@ -45,5 +44,5 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
If this is going to work, you have to change the version number, so you may just upgrade to the latest version